flake.nix

```diff
@@ -249,6 +270,29 @@
             mkdir $out # success
           '';
 
+          checks.age-plugin = pkgs.runCommand "age-plugin" { } ''
+            set -euo pipefail
+            cp -r '${./.}/example/.' "$TMPDIR"
+            cd "$TMPDIR/plugin-example"
+
+            PATH=$PATH:${exampleRagePlugin}/bin
+
+            # Encrypt with ragenix
+            echo wurzelpfropf | ${pkgs.ragenix}/bin/ragenix --editor - --edit unencrypted.age
+
+            # Decrypt with rage
+            decrypted="$(${pkgs.rage}/bin/rage -i '${./example/keys/example_plugin_key.txt}' -d unencrypted.age)"
+            if [[ "$decrypted" != "wurzelpfropf" ]]; then
+              printf 'Unexpected value for decryption with plugin'
```

Comment:
Either use `printf` or `echo`, consistently. For printing variables without any expansion and substitution, it's probably more canonical to use `echo` (IMHO).